Sources/Server/Response.swift

```diff
@@ -57,6 +57,12 @@ public class Response {
         let header = buildHeader(statusCode: status200, contentLength: body.utf8.count)
         return header + body
     }
+
+    private func handleRedirect(request: Request) -> String {
+        let body = prepareBody(body: "Hello World", method: request.method)
+        let header = "HTTP/1.1 302\r\nContent-Length: \(body.utf8.count)\r\nContent-type: text/html\r\nLocation: /\r\n\r\n" 
```

Comment:
Why is the `Location` header blank?

Reply:
There is a root path assigned that gets lost in the slashes. `Location: /`

Reply:
Ah. I see that now.